crates/aiken-project/src/lib.rs

```diff
@@ -464,6 +466,27 @@ where
         })
     }
 
+    pub fn export(&self, module: &str, name: &str) -> Result<Export, Error> {
+        self.checked_modules
+            .get(module)
+            .and_then(|checked_module| {
+                checked_module.ast.definitions().find_map(|def| match def {
+                    Definition::Fn(func) if func.name == name => Some((checked_module, func)),
+                    _ => None,
+                })
+            })
+            .map(|(checked_module, func)| {
+                let mut generator = self.new_generator(Tracing::silent());
```

Comment:
What about allowing tracing options on export? One might want to export with or without traces. If not, I think the most sensible default then is to export _with traces_ as the main usage intent for this command is for testing and debugging.

Reply:
I can allow the flag to go through